### DISCUSSION TITLE
Precount tempo disappear in Musescore4?

### DISCUSSION THREAD

**Discussion by benoitnicomette** [musescore/MuseScore#16425] [Q&A]
Hello Musescore Community.

I can't find precount tempo in Musescore4?

Can you help me to find it please?

Have a nice day

Benoit (France)

**Comment by MarcSabatella** [comment]
If you mean the count-in that one could hear before playback, that option is not currently present, but has been requested and will hopefully make it back at some point.

In the future, please use the official Support forum on musescore.org when asking for help using MuseScore.

**Comment by benoitnicomette** [comment]
Thank you very much Marc.

Have a nice day
